test/book/cornerstones/groups/actions.jlcon

```diff
@@ -60,10 +60,10 @@ Group homomorphism
 julia> isomorphism(PermGroup, U)
 Group homomorphism
   from pc group of order 8
-  to permutation group of degree 8 and order 8
+  to permutation group of degree 4 and order 8
```

Comment:
This example was originally intended to demonstrate that `isomorphism` is in general *not* good enough to get such a small degree for the image. Now this part of the book chapter becomes less suggestive.
(Since the final proofs of the Oscar book are still not available, do we have a chance to replace this example *in the book* by a better one? Of course the next release of GAP may then destroy the effect again ...)